diff --git a/node_modules/rfc4648/lib/src/index.d.ts b/node_modules/rfc4648/lib/src/index.d.ts
index 438af9a..de74fa6 100644
--- a/node_modules/rfc4648/lib/src/index.d.ts
+++ b/node_modules/rfc4648/lib/src/index.d.ts
@@ -1,4 +1,4 @@
-import { parse, ParseOptions, stringify, StringifyOptions } from './codec';
+import { parse, ParseOptions, stringify, StringifyOptions } from './codec.js';
 export declare const base16: {
     parse(string: string, opts?: ParseOptions | undefined): Uint8Array;
     stringify(data: ArrayLike<number>, opts?: StringifyOptions | undefined): string;
